ygen/ir.go

```diff
@@ -364,10 +403,11 @@ type NodeDetails struct {
 	// Shadow paths are paths that have sibling config/state values
 	// that have been compressed out due to path compression.
 	ShadowMappedPathModules [][]string
-	// TODO(wenbli): Think about how to add this in an easily-usable way.
+	// LangMapper during IR generation to assist the code generation stage.
```

Comment:
Not sure this makes sense -- the map[string]string needs to implement the interface? Could you clarify the comment?

Reply:
Fixed comment.

Reply:
Thanks, took me some time to find out how to see that in the GitHub UI, but LGTM.